connector/settings.gradle.kts

```diff
@@ -17,4 +17,6 @@ dependencyResolutionManagement {
 include(":launchers:controlplane")
 include(":launchers:dataplane")
 include(":launchers:runtime-embedded")
+include(":launchers:api")
```

Comment:
nit: I suggest to keep dependencies in alphabetical order, for ease of reference.